### ISSUE TITLE
docker-compose Bug on Mac

### ISSUE THREAD

**Issue by beeankha** [ansible/awx#2505]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME

 - docker compose development environment

##### ENVIRONMENT
* AWX version: 2.0.1
* AWX install method: Docker version 18.06.1-ce, build e68fc7a
* Ansible version:  2.6.0
* Operating System:  MacOS High Sierra 10.13.6
* Web Browser: Chrome

##### STEPS TO REPRODUCE

Followed the instructions listed here: 
https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md#setting-up-your-development-environment 

After executing the `make docker-compose` step, I get the following permission errors:

```
awx_1        | ln: failed to create symbolic link ?/venv/awx/lib/python2.7/site-packages/rdb.py?: Permission denied
awx_1        | cp: cannot create regular file ?/supervisor.conf?: Permission denied
awx_1        | 20:41:31 nginx.1      | nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (13: Permission denied)
awx_1        | 20:41:31 nginx.1      | 2018/10/19 20:41:31 [emerg] 306#0: mkdir() "/var/lib/nginx/tmp/client_body" failed (13: Permission denied)
...
tools_awx_1 exited with code 2 
```

To inspect what was going on, started the container with `make docker-compose-test`, but
`/supervisor.conf` does not exist
```
[root@awx /]# ls -al /superviser.conf
ls: cannot access /superviser.conf: No such file or directory
[root@awx /]# ls -al /var/lib/nginx/tmp/
total 8
drwx------ 2 nginx nginx 4096 Mar  6  2018 .
drwx------ 3 nginx nginx 4096 Oct 16 13:48 ..
```

The user being used does not seem to have permission to files it needs when starting process.

##### EXPECTED RESULTS

To be able to successfully run development environment with docker-compose on macOS using docker for mac

##### ACTUAL RESULTS

`awx` user in container does not have sufficient permissions to create necessary files

##### ADDITIONAL INFORMATION

https://gist.github.com/beeankha/0bfd17c27571b69c7cc77f5bf3786319

**Comment by wenottingham** [comment]
I don't think this is mac-specific, I've seen similar errors on Linux.

**Comment by matburt** [comment]
This will be fixed with #2508
